config/default.go

```diff
@@ -45,6 +45,9 @@ test:
 record:
   recordTimer: 0s
   filters: []
+contract:
+  servicesMapping: {"s2":["/s2endpoint","localhost:9222"],"s3":["/s3endpoint","/otp-service/"],"s5":["localhost:3000","/endpoint"]}
+  self: "s1"
```

Comment:
`driven` flag field should also be here because it is a user-facing field, not just for debugging purposes.
and set its default value as `consumer`.